cmd2/cmd2.py

```diff
@@ -720,23 +720,20 @@ def _reset_completion_defaults(self) -> None:
             readline.rl.mode._display_completions = self._display_matches_pyreadline
 
     def tokens_for_completion(self, line: str, begidx: int, endidx: int) -> Tuple[List[str], List[str]]:
-        """
-        Used by tab completion functions to get all tokens through the one being completed
+        """Used by tab completion functions to get all tokens through the one being completed.
+
         :param line: the current input line with leading whitespace removed
         :param begidx: the beginning index of the prefix text
         :param endidx: the ending index of the prefix text
         :return: A 2 item tuple where the items are
-                 On Success
-                     tokens: list of unquoted tokens
-                             this is generally the list needed for tab completion functions
-                     raw_tokens: list of tokens with any quotes preserved
-                                 this can be used to know if a token was quoted or is missing a closing quote
-
-                     Both lists are guaranteed to have at least 1 item
-                     The last item in both lists is the token being tab completed
-
-                 On Failure
-                    Two empty lists
+                 **On Success**
+                 - tokens: list of unquoted tokens - this is generally the list needed for tab completion functions
```

Comment:
The `autodoc` extension for Sphinx is rather particular when it comes to indentation within the params/return section of docstrings.  If we want to make it look pretty in the docs, we can use full reStructuredText however.

All changes made in **cmd2.py** were made specifically to fix `autodoc` build errors.